.github/workflows/sub_essential_tests.yml

```diff
@@ -69,6 +69,7 @@ jobs:
           SEP1_TOML_VALUE: /home/runner/java-stellar-anchor-sdk/service-runner/src/main/resources/config/stellar.host.docker.internal.toml
           SEP10_WEB_AUTH_DOMAIN: host.docker.internal:8080
           SEP10_HOME_DOMAINS: host.docker.internal:8080,*.stellar.org
+          TOML_PATH: /home/runner/java-stellar-anchor-sdk/wallet-reference-server/src/main/resources/assets
```

Comment:
Is this necessary? I don't see other `stellar.toml` for the wallet reference server?

Reply:
Yes it is. Otherwise it can't read toml the way we package the jar right now